CSharp/Library/Microsoft.Bot.Builder.Azure/SqlBotDataStore.cs

```diff
@@ -165,6 +165,26 @@ internal class SqlBotDataContext : DbContext
         public SqlBotDataContext(string connectionString)
             : base(connectionString)
         {
+            Database.SetInitializer<SqlBotDataContext>(null);
+        }
+
+        /// <summary>
+        /// Throw if the database or SqlBotDataEntities table have not been created.
+        /// </summary>
+        static internal void AssertDatabaseReady()
```

Comment:
Why not private?

Reply:
It is called from AzureModule, during autofac registrations.  Maybe the method should be in AzureModule?